site.yml

```diff
@@ -9,9 +9,9 @@ content:
     - HEAD
   - url: https://github.com/owncloud/docs.git
     branches:
+    - '10.4'
     - '10.3'
     - '10.2'
-    - '10.1'
```

Comment:
Same here, keep it as part of the list.

Reply:
As far I remember (and I may be wrong) that only that latest three releases should be kept?

Reply:
These versions will receive backports, this doesn't mean that it will be removed from the version listing

Reply:
ok, thanks for the clarification.
Q: is there any statement how many releases are kept and when rolling starts? eg keep all 10.x releases until 11. And in case, which of the 10 release would get phased out.

Reply:
@tboerger, customer support made the decision to only backport changes to the previous version. The previous _three_ versions, however, do get updates if they're to correct severe bugs. Otherwise, it's just the last version that's updated.

Reply:
Why should they be dropped from the version list at all?

Reply:
Because the decision was also made to only link the current version and the two previous versions in the documentation. I'm just going along with it.

Reply:
10.1 has been re-added to the list of available content sources.